deterministic.c

```diff
@@ -5,14 +5,20 @@
 #include "inner.h"
 #include "deterministic.h"
 
+#define FALCON_DET1024_TMPSIZE_KEYGEN FALCON_TMPSIZE_KEYGEN(FALCON_DET1024_LOGN)
+#define FALCON_DET1024_TMPSIZE_SIGNDYN FALCON_TMPSIZE_SIGNDYN(FALCON_DET1024_LOGN)
+#define FALCON_DET1024_TMPSIZE_VERIFY FALCON_TMPSIZE_VERIFY(FALCON_DET1024_LOGN) 
+#define FALCON_DET1024_SALTED_SIG_COMPRESSED_MAXSIZE FALCON_SIG_COMPRESSED_MAXSIZE(FALCON_DET1024_LOGN)
+#define FALCON_DET1024_SALTED_SIG_CT_SIZE FALCON_SIG_CT_SIZE(FALCON_DET1024_LOGN)
```

Comment:
are you kind of intending to have these perform like global const variables?

Reply:
no. I don't them to be global. I wanted to use some defines to have a const expression to define arrays on stack.
do you see any other option? (I can always use FALCON_DET1024_LOGN. i think it less readable)

Reply:
wdyt of this though?
```
const size_t FALCON_DET1024_TMPSIZE_KEYGEN = FALCON_TMPSIZE_KEYGEN(FALCON_DET1024_LOGN);
const size_t FALCON_DET1024_TMPSIZE_SIGNDYN = FALCON_TMPSIZE_SIGNDYN(FALCON_DET1024_LOGN);
const size_t FALCON_DET1024_TMPSIZE_VERIFY = FALCON_TMPSIZE_VERIFY(FALCON_DET1024_LOGN);
const size_t FALCON_DET1024_SALTED_SIG_COMPRESSED_MAXSIZE = FALCON_SIG_COMPRESSED_MAXSIZE(FALCON_DET1024_LOGN);
const size_t FALCON_DET1024_SALTED_SIG_CT_SIZE = FALCON_SIG_CT_SIZE(FALCON_DET1024_LOGN);
```

Reply:
sorry
you get the same warning on that

Reply:
I guess that is still a VLA. well it seems fine! I guess I should not worry about the C preprocessor evaluating macro expressions and cgo, it is just magic that works.